MediaBrowser.Model/Entities/MediaStream.cs

```diff
@@ -137,7 +149,8 @@ public string DisplayTitle
                     {
                         var attributes = new List<string>();
 
-                        if (!string.IsNullOrEmpty(Language))
+                        // Do not display the language code in display titles if unset or set to a special code. Show it in all other cases (possibly expanded).
+                        if (!string.IsNullOrEmpty(Language) && !_specialCodes.Contains(Language, StringComparer.OrdinalIgnoreCase))
```

Comment:
We have our own extension method that is slightly better optimized and uses StringComparison (may require an additional `using`)
```suggestion
                        if (!string.IsNullOrEmpty(Language) && !_specialCodes.Contains(Language, StringComparison.OrdinalIgnoreCase))
```